### ISSUE TITLE
`image-set` doesn't worked with `cursor`.

### ISSUE THREAD

**Issue by HyunSeob** [postcss/autoprefixer#899]
`image-set` works well with `background-image` or `background`, but doesn't work with `cursor` as you can see below.

Input: 
``` css
.background {
  background: image-set('foo.png' 1x, 'foo-2x.png' 2x);
}

.cursor {
  cursor: image-set('foo.png' 1x, 'foo-2x.png' 2x);
}
```

Output:
``` css
.background {
  background: -webkit-image-set(url('foo.png') 1x, url('foo-2x.png') 2x);
  background: image-set('foo.png' 1x, 'foo-2x.png' 2x);
}

.cursor {
  cursor: image-set('foo.png' 1x, 'foo-2x.png' 2x);
}
```

**Comment by ai** [comment]
I will look at it in the weekend.

**Comment by ai** [comment]
Done c6a6237

**Comment by ai** [comment]
Released in 7.1.4
